### ISSUE TITLE
Schema suggestions

### ISSUE THREAD

**Issue by BobRay** [devtrench/WordPress-to-MODx#1]
James, It doesn't look like much is going on with this, but I thought I'd mention something about the schema in case you pick it up again.

The class names in the schema should be singular nouns, e.g. Post, User, Comment. Each one represents a single object when used in code as in: $modx->getObject('Post'). More importantly, you'll want to use the plural version of the names as related objects so you can do stuff like this:

// all posts of one user:

$posts = $user->getMany('Posts');

// all comments on one post:

$post->getMany('Comments);

Using the plural versions for the class names as you currently do will make the code confusing and will foreclose the option to use the plural versions as related object aliases.

$user->getMany

**Comment by devtrench** [comment]
Yeah, I've been very busy and am not sure if I can get back to this.  Thanks for making the suggestions :)
